geoportailv3/static/js/profile/profiledirective.js

```diff
@@ -226,6 +226,8 @@ app.ProfileController = function($scope) {
             p.getCoordinates().concat(newVal[i]['dist']));
       }
       this.line_ = lineString;
+    } else {
+      delete this.line_;
```

Comment:
I'd rather initialize `this.line_` to `null` in the constructor, and reset it to `null` here.

```js
  /**
   * @type {ol.geom.LineString}
   * @private
   */
  this.line_ = null;
```